src/docs/10_quality_scenarios.adoc

```diff
@@ -36,6 +36,8 @@ The quality tree is a high-level overview of the quality goals and requirements:
 In any case the tree should include links to the scenarios of the following section.
 ****
 
+= image:qualityTree.jpg[Quality tree] 
```

Comment:
I think to add images in ascii doc you should use "image:: "

Reply:
It was:
=image:images/image.jpg[altText]
Thanks!